Traceback (most recent call last):
  File "D:/Users/Admin/PycharmProjects/PyCharm_Projects/Deep_learning/cat_dog/model_training02.py", line 98, in <module>
    model = LeNet.LeNet("LeNet")
  File "D:\Users\Admin\PycharmProjects\PyCharm_Projects\Deep_learning\cat_dog\LeNet.py", line 32, in __init__
    self.fc2 = paddle.nn.Linear(in_features=64, out_features=num_classes)
  File "D:\Users\Admin\Python37\lib\site-packages\paddle\nn\layer\common.py", line 119, in __init__
    is_bias=False)
  File "D:\Users\Admin\Python37\lib\site-packages\paddle\fluid\dygraph\layers.py", line 412, in create_parameter
    default_initializer)
  File "D:\Users\Admin\Python37\lib\site-packages\paddle\fluid\layer_helper_base.py", line 374, in create_parameter
    **attr._to_kwargs(with_initializer=True))
  File "D:\Users\Admin\Python37\lib\site-packages\paddle\fluid\framework.py", line 2882, in create_parameter
    param = ParamBase(*args, **kwargs)
  File "D:\Users\Admin\Python37\lib\site-packages\decorator.py", line 232, in fun
    return caller(func, *(extras + args), **kw)
  File "D:\Users\Admin\Python37\lib\site-packages\paddle\fluid\wrapped_decorator.py", line 25, in __impl__
    return wrapped_func(*args, **kwargs)
  File "D:\Users\Admin\Python37\lib\site-packages\paddle\fluid\framework.py", line 227, in __impl__
    return func(*args, **kwargs)
  File "D:\Users\Admin\Python37\lib\site-packages\paddle\fluid\framework.py", line 5499, in __init__
    if each < 0:
TypeError: '<' not supported between instances of 'str' and 'int'
开始训练...
训练集数据处理。。。。
=============
验证集数据处理。。。。。
=============
0
epoch: 0, batch_id: 0, loss is：0.8431
[validation] accuracy 0.6250, loss 0.3954
1
epoch: 1, batch_id: 0, loss is：0.3453
[validation] accuracy 0.6250, loss 0.1724
2
epoch: 2, batch_id: 0, loss is：0.1440
[validation] accuracy 0.8000, loss 0.0881
3
epoch: 3, batch_id: 0, loss is：0.0834
[validation] accuracy 0.8750, loss 0.0574
4
epoch: 4, batch_id: 0, loss is：0.0580
[validation] accuracy 1.0000, loss 0.0435
5
epoch: 5, batch_id: 0, loss is：0.0477
[validation] accuracy 1.0000, loss 0.0358
6
epoch: 6, batch_id: 0, loss is：0.0399
[validation] accuracy 1.0000, loss 0.0309
7
epoch: 7, batch_id: 0, loss is：0.0344
[validation] accuracy 1.0000, loss 0.0274
8
epoch: 8, batch_id: 0, loss is：0.0307
[validation] accuracy 1.0000, loss 0.0247
9
epoch: 9, batch_id: 0, loss is：0.0279
[validation] accuracy 1.0000, loss 0.0225
D:\Users\Admin\Python37\lib\site-packages\paddle\nn\layer\norm.py:641: UserWarning: When training, we now always track global mean and variance.
  "When training, we now always track global mean and variance.")
